test/ERC20Channel.test.js

```diff
@@ -0,0 +1,130 @@
+import latestTime from './helpers/latestTime';
+import { increaseTimeTo } from './helpers/increaseTime';
+var ethUtils = require('ethereumjs-util');
+
+var BigNumber = web3.BigNumber;
+
+require('chai')
+  .use(require('chai-bignumber')(BigNumber))
+  .should();
+
+const assertRevert = require('./helpers/assertRevert');
+
+var StandardTokenMock = artifacts.require('./mocks/StandardTokenMock.sol');
+var ERC20Channels = artifacts.require('./token/ERC20Channels.sol');
+var ECRecovery = artifacts.require('./ECRecovery.sol');
+
+contract('ERC20Channels', function (accounts) {
```

Comment:
Could you add tests for trying to close a channel (both cooperatively and uncooperatively) with a higher balance than the original deposit, _with the contract actually having that balance at its disposal_ (due to other channels being open as well)?

Reply:
yep!

Reply:
Actually, if we split the channels so there is one contract per channel, this test would no longer be needed, right?